webui.py

```diff
@@ -1036,22 +1036,21 @@ def simple_body(self, path):
             raise NotFound, path + " does not have any releases"
 
         html = []
-        html.append("""<html><head><title>Links for %s</title><meta name="api-version" value="2" /></head>"""
+        html.append("""<!DOCTYPE html><html><head><title>Links for %s</title></head>"""
                     % cgi.escape(path))
         html.append("<body><h1>Links for %s</h1>" % cgi.escape(path))
-        for href, rel, text in urls:
+        for href, rel, text, requires_python in urls:
             if href.startswith('http://cheeseshop.python.org/pypi') or \
                     href.startswith('http://pypi.python.org/pypi') or \
                     href.startswith('http://www.python.org/pypi'):
                 # Suppress URLs that point to us
                 continue
-            if rel:
```

Comment:
If I remember correctly, removing `rel='internal'` tag will break older versions of `pip`. I'm not opposed, but it is asking for pitchforks.